src/pdfs/cmeanvar_vmf.jl

```diff
@@ -0,0 +1,67 @@
+export CMeanVarVMF
```

Comment:
I think we should rename this to `CMeanConcVMF`? same goes for the filename

Reply:
Yep, let's rename that. I am sorry.